### PR TITLE
QUICK-FIX Add Audit's ACL roles model for selenium framework

### PR DESCRIPTION
**Pull request by AleksNeStu**

### CODE REVIEW

**Comment by AleksNeStu** [comment]
@SergeyRabtsau or @dnovikau or @ch-t could you please review this one PR.

File: test/selenium/src/lib/entities/entities_factory.py [hunk 1]
```diff
@@ -209,10 +201,7 @@ def generate_ca_defenitions_for_asmt_tmpls(cls, list_ca_definitions):
             for ca_def in list_ca_definitions]
 
   @classmethod
-  def create(cls, title=None, id=None, href=None, type=None,
-             definition_type=None, attribute_type=None, helptext=None,
-             placeholder=None, mandatory=None, multi_choice_options=None,
-             updated_at=None, modified_by=None, created_at=None):
+  def create(cls, **attrs):
```
Comment (ch-t) [inline test/selenium/src/lib/entities/entities_factory.py:204]: really good!

File: test/selenium/src/lib/entities/entities_factory.py [hunk 2]
```diff
@@ -222,12 +211,7 @@ def create(cls, title=None, id=None, href=None, type=None,
     """
     ca_entity = cls._create_random_ca()
     ca_entity = cls._update_ca_attrs_values(
-        obj=ca_entity, is_allow_none_values=False, title=title, id=id,
-        href=href, type=type, definition_type=definition_type,
-        attribute_type=attribute_type, helptext=helptext,
-        placeholder=placeholder, mandatory=mandatory,
-        multi_choice_options=multi_choice_options, updated_at=updated_at,
-        modified_by=modified_by, created_at=created_at)
+        obj=ca_entity, is_allow_none_values=False, **attrs)
```
Comment (ch-t) [inline test/selenium/src/lib/entities/entities_factory.py:214]: as previous, good!

**Review by ch-t — APPROVED** [review]
LGTM!

File: test/selenium/src/lib/entities/entities_factory.py (hunk 2 above)
Comment (AleksNeStu) [reply test/selenium/src/lib/entities/entities_factory.py:214]: I have a draft with a new model ready to ACL and RBAC and with different types of improvements! Changes coming soon:)

**Pull request merged**